### PR TITLE
[Release] Release of vizro-ai 0.3.0

### PR DESCRIPTION
**Pull request by lingyielia**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-ai/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+- VizroAI now has a more flexible output when choosing `VizroAI.plot(...,return_elements=True)`. See documentation for all new options. ([#646](https://github.com/mckinsey/vizro/pull/646))
+- VizroAI now support text to dashboard generation using `VizroAI.dashboard()`. To get started, visit the [Vizro-AI docs](https://vizro.readthedocs.io/projects/vizro-ai/en/latest/pages/tutorials/quickstart-dashboard/)
+
+## Removed
+
+- Removed the automatic display of chart explanation and insights in Jupyter. ([#646](https://github.com/mckinsey/vizro/pull/646))
+
+## Changed
+
+- Changed the return type of `VizroAI.plot(...,return_elements=True)` from `PlotOutputs` dataclass to a pydantic model with more flexible methods. See documentation for more into ([#646](https://github.com/mckinsey/vizro/pull/646))
```
Comment (petar-qb) [inline vizro-ai/CHANGELOG.md:30]: Just double check the `.` usage below. I added 3 `.` suggestions.

```suggestion
## Highlights ✨

- VizroAI now allows **any** `langchain` model with structured output capabilities to be used, not just `ChatOpenAI`. ([#646](https://github.com/mckinsey/vizro/pull/646))
- VizroAI now has a more flexible output when choosing `VizroAI.plot(...,return_elements=True)`. See documentation for all new options. ([#646](https://github.com/mckinsey/vizro/pull/646))
- VizroAI now support text to dashboard generation using `VizroAI.dashboard()`. To get started, visit the [Vizro-AI docs](https://vizro.readthedocs.io/projects/vizro-ai/en/latest/pages/tutorials/quickstart-dashboard/).

## Removed

- Removed the automatic display of chart explanation and insights in Jupyter. ([#646](https://github.com/mckinsey/vizro/pull/646))

## Changed

- Changed the return type of `VizroAI.plot(...,return_elements=True)` from `PlotOutputs` dataclass to a pydantic model with more flexible methods. See documentation for more. ([#646](https://github.com/mckinsey/vizro/pull/646))
```

**Review by petar-qb — APPROVED** [review]
A few `.` suggestions, but this looks good 👍

File: vizro-ai/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.3.0'></a>
+
+# 0.3.0 — 2024-10-02
+
+## Highlights ✨
+
+- VizroAI now allows **any** `langchain` model with structured output capabilities to be used, not just `ChatOpenAI`. ([#646](https://github.com/mckinsey/vizro/pull/646))
+- VizroAI now has a more flexible output when choosing `VizroAI.plot(...,return_elements=True)`. See documentation for all new options. ([#646](https://github.com/mckinsey/vizro/pull/646))
```
Comment (huong-li-nguyen) [inline vizro-ai/CHANGELOG.md:21]:
```suggestion
- VizroAI now has a more flexible output when choosing `VizroAI.plot(...,return_elements=True)`. See documentation for all new options. ([#646](https://github.com/mckinsey/vizro/pull/646))
```

Can we link to the relevant documentation link here?

File: vizro-ai/CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+- VizroAI now support text to dashboard generation using `VizroAI.dashboard()`. To get started, visit the [Vizro-AI docs](https://vizro.readthedocs.io/projects/vizro-ai/en/latest/pages/tutorials/quickstart-dashboard/).
+
+## Removed
+
+- Removed the automatic display of chart explanation and insights in Jupyter. ([#646](https://github.com/mckinsey/vizro/pull/646))
+
+## Changed
+
+- Changed the return type of `VizroAI.plot(...,return_elements=True)` from `PlotOutputs` dataclass to a pydantic model with more flexible methods. See documentation for more info. ([#646](https://github.com/mckinsey/vizro/pull/646))
```
Comment (huong-li-nguyen) [inline vizro-ai/CHANGELOG.md:30]: Here again - add documentation link directly, so they don't have to look?

**Review by maxschulz-COL — APPROVED** [review]

**Pull request merged**